### ISSUE TITLE
PKIX certificate path error

### ISSUE THREAD

**Issue by Nicd** [Nicd/code-stats-intellij#3]
On some rare configurations, the plugin may fail with a PKIX error, complaining it cannot build a valid certification path to the LE certificate. I have as of yet been unable to reproduce this bug. The plugin inserts the DST root CA dynamically to the trust store when run, but that does not seem to help on some machines.

The workaround is to install the CA manually (roughly like this):

```
$ cd /Applications/<IDE>.app/Contents/jre/jdk/Contents/Home/jre
$ bin/keytool -import -alias 'DST Root CA X3' -keystore lib/security/cacerts -trustcacerts -file /path/to/dst_root_ca_x3.pem
```

But this should be fixed in the plugin, if possible.

**Comment by wenom64** [comment]
I have this problem on Windows 10. I have no idea how to fix it.

**Comment by Nicd** [comment]
@wenom64 can you download and install the 1.0.2 version and see if it fixes your issue? It's available here: http://www.jottacloud.com/p/nicd/cc80f8d571ed479194bef633213df31e

To install, go to the IDE's settings -> Plugins -> Install plugin from disk

**Comment by wenom64** [comment]
Cool, now everything works fine. Bug is gone. Thank You :-)

**Comment by Nicd** [comment]
I have released 1.0.3 in the official repos.

**Comment by rosstroha** [comment]
I still had problems, but I figured it out by running this command:
`keytool -importcert -alias "codestats.net" -keystore "C:\Program Files\Java\jdk1.8.0_92\jre\lib\security\cacerts" -storepass changeit -file "C:\Users\${USERNAME}\Desktop\stats_cert2.cer"`

stats_cert2.cer is the certificate I saved through Chrome. To save the cert go to codestats.net and click on the green lock near the URL. Click details near the top. Click View Certificate, then click the Details tab. Copy to file and select Base-64 encoded.

The only problem is that the cert will be expired in the future and you'll need to re-import it.
